ui/raidboss/data/03-hw/alliance/dun_scaith.js

```diff
@@ -430,8 +430,8 @@
         cn: '躲避AOE后击杀康拉',
       },
     },
-    // These triggers are common to both Scathach and Diabolos
     {
+      // This trigger is common to both Scathach and Diabolos, since handling is 100$ identical.
```

Comment:
Sorry, did you mean `100%` here and not `100$`?

Reply:
So I did. Just as well, I caught a couple of other things that needed editing/removing while fixing that.